com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/Nodes/EmissionNode.cs

```diff
@@ -99,13 +99,7 @@ public void GenerateNodeCode(ShaderStringBuilder sb, GenerationMode generationMo
             if (intensityUnit == EmissiveIntensityUnit.EV100)
                 intensityValue = "ConvertEvToLuminance(" + intensityValue + ")";
 
-            sb.AppendLine("#ifdef SHADERGRAPH_PREVIEW");
-            sb.AppendLine($"$precision inverseExposureMultiplier = 1.0;");
-            sb.AppendLine("#else");
-            sb.AppendLine($"$precision inverseExposureMultiplier = GetInverseCurrentExposureMultiplier();");
-            sb.AppendLine("#endif");
-
-            sb.AppendLine(@"$precision3 {0} = {1}({2}.xyz, {3}, {4}, inverseExposureMultiplier);",
+            sb.AppendLine(@"$precision3 {0} = {1}({2}.xyz, {3}, {4});",
```

Comment:
Moved this value inside of the node function, as otherwise it potentially conflicts if you have two Emission nodes in your graph.